rep-2000.rst

```diff
@@ -391,6 +393,8 @@ Dependency Requirements:
 +-------------+----------------+---------------+----------------+---------------------+
 | Connext DDS |                              5.3.1              |         N/A         |
 +-------------+----------------+---------------+----------------+---------------------+
+| Cyclone DDS |                              0.1.0                                    |
```

Comment:
Maybe add a note that the version number is not correct and needs to be determined? Same below.

Reply:
@dirk-thomas updated Cyclone DDS to [0.5.0 release](https://projects.eclipse.org/projects/iot.cyclonedds/releases/0.5.0-eusebius) planned for Nov 8th